RNA-seq/README.md

```diff
@@ -15,3 +15,11 @@ The notebooks that comprise this module are (estimated time to complete notebook
 
 
 _Total estimated time to complete instruction notebooks: 3 hours 15 minutes_
+
+Beyond the notebooks mentioned above, this module has the following structure:
+
+![module project structure](./diagrams/bulk_project_structure.png)
+
+With the format of each subdirectory in `data` being similar to:
+
+![gastric cancer structure](./diagrams/gastric_cancer_structure.png)
```

Comment:
Just wanted to note that this still contains a reference to `tximport` which we should fix.

Reply:
Ah, I see now - apologies!

> but was not sure if this update should be propagated to the comments on the gastric_cancer_structure.png annotated figure since my understanding is that tximeta does indeed implement tximport

Reply:
No worries! Good to note your stance that it should indeed be updated 👍